packages/suite/src/components/suite/Skeleton/index.tsx

```diff
@@ -0,0 +1,94 @@
+import styled, { css, keyframes } from 'styled-components';
+
+interface StackProps {
+    col?: boolean;
+    grow?: boolean;
+    margin?: string;
+    childMargin?: string;
+    alignItems?: string;
+}
+const Stack = styled.div<StackProps>`
+    display: flex;
+    align-items: ${props => props.alignItems ?? 'auto'};
+    flex-direction: ${props => (props.col ? 'column' : 'row')};
+    flex-grow: ${props => (props.grow ? 1 : 0)};
+    ${props =>
+        props.margin &&
+        css`
```

Comment:
I'd say it is needed. Otherwise you would need to provide some default value for margin and I don't know which one you would chose.

Reply:
I would say it is essential, you can get some auto-completion hints with it, otherwise you re basically setting dummy template string and linters/vscode/whatever will treat it like.
<img width="427" alt="Screenshot 2020-12-04 at 12 08 50" src="https://user-images.githubusercontent.com/6961901/101156835-7a660f00-3629-11eb-909a-28b5cc3087ba.png">


<img width="709" alt="Screenshot 2020-12-04 at 12 07 44" src="https://user-images.githubusercontent.com/6961901/101156732-530f4200-3629-11eb-99e6-1572d0295dcd.png">

Reply:
oh, now I noticed that @mroz22  and I have different understanding of your comment 😁

Reply:
if @mroz22 got this right, then yes basically I don't want to generate margin property if there is no margin passed to the component (saving that precious bytes we always care about! 😁 )

Reply:
actually both of you are right :)